src/gui/macutilities.h

```diff
@@ -0,0 +1,13 @@
+#ifndef MACUTILITIES_H
```

Comment:
You should specify the email after your name. You don't want to do it because of some reasons?